Draft quick-start at beginning of user-guide (#108)
smash-transport/smash-vhlle-hybrid: docs/user/index.md

[^1]: Be aware, that this might not work straightaway if only a very old Bash installation is available on your OS.
      However, if your Bash version is `4.x` or higher, the main script will be able to give you a complete overview of system requirements.

## I wanna run the tool **NOW** – what should I do?

!!! warning "Sure, give it a try, but with awareness &ensp; :four_leaf_clover:"

    Reading (part of) the user guide is encouraged not only to get acquainted with how the Hybrid-handler works, but also to discover what it has to offer you. Here in the following you find a very minimal quick-start, that might work out straightforwardly, but that might also fail depending on your setup. Please, refer to the user guide for further information.

??? note "The very first steps"

    ```bash
    $ git clone https://github.com/smash-transport/smash-vhlle-hybrid.git
    # some standard output (1)
    $ ./smash-vhlle-hybrid/Hybrid-handler
    # A nice, colorful report about system requirements (2)
    ```

    1. Git will clone using HTTPS protocol and locally create a :file_folder: ***smash-vhlle-hybrid*** folder.

>     2. :warning: At this point you will discover whether your OS is ready to go. If it is, you will see green version numbers and green ticks :white_check_mark: as in the image shown here below. If you see some red version number or red cross :x:, you need to take action. ![Image title](../images/Helper-example.png)

Reviewer: ```suggestion
    2. :warning: At this point you will discover whether your OS is ready to go. If it is, you will see green version numbers and green ticks :white_check_mark: as in the image shown here below. If you see some red version number or red cross :x:, you need to install or update the corresponding requirement. ![Image title](../images/Helper-example.png)
```

> ??? abstract "Once the OS is ready"

Reviewer: ```suggestion
??? abstract "Setting up once the OS is ready"
```

    ```bash hl_lines="5 6"
    # Make sure all the software to be installed is compiled/installed (1)
    $ mkdir my-first-hybrid-run
    $ cp smash-vhlle-hybrid/configs/predef_configs/config_TEST.yaml my-first-hybrid-run/hybrid.yaml
    $ cd my-first-hybrid-run
    $ SMASH_EXE='/path/to/smash'; HYDRO_EXE='/path/to/vHLLE'; SAMPLER_EXE='/path/to/hadron-sampler' # (2)!
    $ sed -i -e "s|/path/to/smash|${SMASH_EXE}|g" -e "s|/path/to/vHLLE|${HYDRO_EXE}|g" -e "s|/path/to/hadron-sampler|${SAMPLER_EXE}|g" hybrid.yaml
    # Optionally check that the replacement has happened correctly
    $ grep 'Executable:' hybrid.yaml # (3)!
    ```

    1. :bulb: You will need to hand over the path to each executable to the Hybrid-handler through its configuration file, which we are setting up next.

>     2. :pencil: You need to replace these three paths to the actual three paths to your software:exclamation:

Reviewer: I think we need only one of these

Author: Since the first is more general and the second more technical, I think it does not harm to have both. I will keep them.

    Alternatively to run these two highlighted lines, you can simply open the :material-file: _hybrid.yaml_ file and specify the paths there by hand at the 4 `Executable:` keys.
    3. This should print the four executable paths in the `IC`, `Hydro`, `Sampler`, `Afterburner` order.

> ??? tip "Finally running!"

Reviewer: ```suggestion
??? tip "Running everything!"
```
